src/home/NotificationCenter.test.tsx

```diff
@@ -122,9 +128,39 @@ const mockcUsdWithoutEnoughBalance = {
   },
 }
 
+const layoutNotificationList = (screen: ReturnType<typeof render>) => {
```

Comment:
this is great!! could we add a comment to remind us that this function is needed to get the onViewableItems callback to fire? 🙏🏻